src/native/src/core/nova_renderer.cpp

```diff
@@ -6,8 +6,8 @@
 
 #include <easylogging++.h>
 
-#include "../gl/windowing/glfw_gl_window.h"
-#include "../utils/utils.h"
+#include "gl/windowing/glfw_gl_window.h"
```

Comment:
I changed all of the relative includes to absolute. from the src directory. Because i cant stand seeing multiple levels of ../../../ in #include statments...